src/views/certificate.ejs

```diff
@@ -47,8 +47,10 @@
                     <div style="font-size: 2.4rem;padding-top: 150px;font-family: 'Roboto Slab', serif;line-height: 55px;">
                         This is to certify that <br/>
                         <span style='color: #e0442c;font-weight: 900;font-size: 3.5rem;'> <%= data.userName%> </span><br/>
-                        has actively contibuted to open source project <b><%= data.projectRepo %></b> of <b><%= data.projectOwner %>.</b>
+                        has actively contibuted to open source project <b><%= data.projectRepo %></b> of <b><%= data.projectOwner %>.</b> 
+                        
                     </div>
+                    <h2><b>Last commit date: </b> <%= data.lastCommitDate %></h2>
                 </div>
             </center> 
             <div class="flex-container">

```

Comment:
As you have added latest commit. reduce the top margin of `flex-container` for proper alignment.